plugins/auth-backend/src/providers/okta/provider.ts

```diff
@@ -89,6 +91,7 @@ export class OktaAuthProvider implements OAuthHandlers {
     this.signInResolver = options.signInResolver;
     this.authHandler = options.authHandler;
     this.resolverContext = options.resolverContext;
+    this.scope = options.scope || 'openid email profile offline_access';
```

Comment:
Alright, so only full replacement. This isn't terrible, but does have the result that users need to remember to duplicate all of these required scopes and we can't increase that default set without breaking those users who picked their own.

Would it be solving your use case to instead have an `additionalDefaultScope` config parameter for now? Then if someone really needs to replace them wholesale, we could add a parameter for that too

Reply:
Yes, it would solve my use case to have a config of additional scopes in addition to the default ones.